application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/reporting/domain/ReportCalculation.kt

```diff
@@ -29,6 +29,7 @@ data class ReportCalculation(
     var status: ReportCalculationStatus,
     var startDate: String? = null,
     var endDate: String? = null,
+    var args: MutableMap<String, String> = mutableMapOf(),
```

Comment:
Are "start" and "end" date here the time period args for filtering data - or calculation run metadata? (maybe add code doc to clarify?)

If the first - should this just merge with args in general?

Reply:
The startDate and endDate are just metadata for the calculation run. The args are the arguments passend in the sql query, so for example an id, or a date range for filtering the entities.